MultiplayerMod/Platform/Steam/Network/Configuration.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using MultiplayerMod.Platform.Steam.Network.Messaging;
+using Steamworks;
+
+namespace MultiplayerMod.Platform.Steam.Network;
+
+public abstract class Configuration {
+
+    public static SteamNetworkingConfigValue_t SendBufferSize(int size = 10485760) => new() {
```

Comment:
Magic number

Reply:
Don't think it deserves a constant, it's just size in bytes by default. `10 * 1024 * 1024` is just weird, idk.

Reply:
Perhaps add a comment about this ? :)